ctsTraffic/ctsTraffic.Config.json

```diff
@@ -11,14 +11,7 @@
     },
 
     "CtsTrafficAzure": {
-        "Iterations" : 1,
-        "StartPort"  : 50000,
-        "tcp" : {
-            "BufferLen"     : [4194304],
-            "Connections"   : [64], 
-            "Options": ""
-        }, 
-        "Runtime"  : 95
+        $null
```

Comment:
Given the recent 100G scaling differences between CTStraffic and NTTTCP TPUT, why not include CTS traffic also as another source of data?